main.tf

```diff
@@ -5,8 +5,19 @@ locals {
     elb_name         = null
     container_port   = var.port_gateway
   }] : []
-  container_cpu         = var.container_cpu != null ? var.container_cpu : data.aws_ssm_parameter.container_cpu[0].value
-  docker_labels         = merge({}, var.docker_labels)
+  container_cpu = var.container_cpu != null ? var.container_cpu : data.aws_ssm_parameter.container_cpu[0].value
+  docker_labels_traefik = {
```

Comment:
this is a generic module and the labels can already be passed in via var.docker_labels. There is no need for this changeset imho.

Reply:
I don't agree as you have quite some dependencies on other configurations/modules. So I think it should not be allowed to have a custom version on every module call.